src/templates/blogListTemplate.tsx

```diff
@@ -171,6 +171,11 @@ export const query = graphql`
     }
   }
 `
+// Explanation on SEO links on this page:
+// According to this sources:
+// 1: https://developers.google.com/search/docs/crawling-indexing/canonicalization
+// 2: https://www.searchenginejournal.com/what-is-a-canonical-url/469636/
+// Best way -- to do nothing
```

Comment:
We'll need a document in `docs/seo.md` to keep SEO information and decisions. Could you move it there and expand on what "do nothing" exactly means?

Reply:
moved and expanded